src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -195,7 +195,9 @@ protected void handleUserException(Exception e) {
         if(printStackTraceOnUserExceptions()) {
             e.printStackTrace();
         } else {
-            System.err.println("Use -D" + STACK_TRACE_ON_USER_EXCEPTION_PROPERTY + "to print the stack trace.");
+            System.err.println("Use the system property "
+                            + STACK_TRACE_ON_USER_EXCEPTION_PROPERTY
+                            + " (--javaOptions '-DGATK_STACKTRACE_ON_USER_EXCEPTION=true') to print the stack trace.");
```

Comment:
@magicDGS this is a good point, thanks for pointing that out.  

Would expanding the comment to say something like "Use the system property STACKTRACE to print a stack trace when user exceptions occur.   If you're running with gatk-launch you can set it with "--javaOptions '-DSTACKTRACE=true".

Would that work for you?  Alternatively, you could override the method in your Main instance.